src/platform/wearables/wearable.ts

```diff
@@ -166,28 +180,7 @@ export namespace Wearable {
    *    - rarity
    *  Third Party Wearables should contain:
    *    - merkleProof
+   *    - content
    */
-
-  export const validate: ValidateFunction<Wearable> = (
-    wearable: any
-  ): wearable is Wearable => {
-    const validationResult =
-      schemaValidator(wearable) &&
```

Comment:
where are these validations running now?

Reply:
They're all implemented as part of ajv validator. Look at line 185, that's where we create the validator. Before we had to keep it in an instance var so we could combine it with the other (manual) checks.